marlowe/README.md

```diff
@@ -10,6 +10,6 @@ by Simon Thompson and Pablo Lamela Seijas
 
 To run tests, from this folder:
 ```bash
-nix run ../.#marlowe.haskell.packages.marlowe.components.tests.marlowe-test
```

Comment:
Unfortunately this is a limitation with nix:
It will `CD` into the folder containing the `flake.nix` and the run the command.
This means that 
```
nix run ../.#marlowe.haskell.packages.marlowe.components.tests.marlowe-test
``` 
Will fail with :
```
reference/dat:  directory not found
```
We must use cabal here

Reply:
Thanks!